packages/rendermime/src/rendermime.ts

```diff
@@ -363,32 +298,56 @@ namespace RenderMime {
      */
     contents: Contents.IManager;
   }
+
+  /**
+   * Add the default renderer factories to a rendermime instance.
+   */
+  export
+  function addDefaultFactories(rendermime: RenderMime): void {
```

Comment:
I thought we were not going to do this, and have the app add the default factories?